pkg/apis/kops/v1alpha1/networking.go

```diff
@@ -52,3 +53,7 @@ type KopeioNetworkingSpec struct {
 // Weave declares that we want Weave networking
 type WeaveNetworkingSpec struct {
 }
+
+// Calico declares that we want Calico networking
+type CalicoNetworkingSpec struct {
+}
```

Comment:
Did you regenerate the api? v1alpha2 is in there https://github.com/kubernetes/kops/blob/master/docs/development/api_updates.md